cylc/flow/scheduler.py

```diff
@@ -1455,7 +1455,7 @@ async def main_loop(self):
             tinit = time()
 
             # Useful for debugging core scheduler issues:
-            # self.pool.log_task_pool(logging.CRITICAL)
+            self.pool.log_task_pool(logging.CRITICAL)
```

Comment:
I assume you'll want to re-comment-out this?
```suggestion
            # self.pool.log_task_pool(logging.CRITICAL)
```